src/components/Pagination/__tests__/__snapshots__/Pagination.spec.js.snap

```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Pagination> that renders a pagination component should render a default pagination 1`] = `
+<nav
+  aria-label="pagination"
+  className="Pagination Pagination--align Pagination--align_center"
+>
+  <Button
+    aria-disabled={true}
+    className="Pagination__button Pagination__button--align Pagination__button--align_center"
+    context="link"
+    data-page={0}
+    disabled={true}
+    isBlock={false}
+    isInline={false}
+    onClick={[Function]}
+    size="normal"
+    type="button"
```

Comment:
@eszthoff Minor detail but something I didn't quite understand myself is why the snapshot contains this `type` attribute since it is not explicitly applied in `Pagination.js` (so the `Button` component would internally use the default, which is `button`, but normally that wouldn't be reflected in the snapshot, or would it?) - Same for the `PaginationButton` snapshot.

Only relevant if you choose to proceed with this implementation of course.

Reply:
AFAIK e.g. `Button.defaultProps` are included in snapshots.